automated install/basic-install.sh

```diff
@@ -258,7 +258,9 @@ if is_command apt-get ; then
     PIHOLE_DEPS=(cron curl dnsutils iputils-ping lsof netcat psmisc sudo unzip wget idn2 sqlite3 libcap2-bin dns-root-data libcap2)
     # The Web dashboard has some that also need to be installed
     # It's useful to separate the two since our repos are also setup as "Core" code and "Web" code
-    PIHOLE_WEB_DEPS=(lighttpd "${phpVer}-common" "${phpVer}-cgi" "${phpVer}-${phpSqlite}" "${phpVer}-xml" "${phpVer}-intl")
+    PIHOLE_WEB_DEPS=(lighttpd "${phpVer}-common" "${phpVer}-cgi" "${phpVer}-${phpSqlite}" "${phpVer}-intl")
+    # - Do not add php5-xml, which does not exist and is compiled into php5-cgi on Debian and Ubuntu
+    [[ ${phpVer} == 'php5' ]] || PIHOLE_WEB_DEPS+=("${phpVer}-xml")
```

Comment:
Break this out to be an `if` block please.

Reply:
What is the benefit of an `if` block in this case? Since it is a check against a literal string, it is 100% failsafe doing what it's made for. Or is it for code style reasons?

Reply:
Code style. One liners are easy to get if you know the language, but we do like to try and have the install script readable by even the most amateur bash scripters.

`if` block just makes it more obvious what is going on.